test/byContract/IexecEscrow/IexecEscrowNative.ts

```diff
@@ -83,13 +83,34 @@ if (CONFIG.chains.default.asset === 'Native') {
                 await expect(iexecPocoAsAccountA.deposit(...depositArgs))
                     .to.changeEtherBalances(
                         [accountA, iexecPoco],
-                        [-depositNativetAmount, depositNativetAmount],
+                        [-depositNativeAmount, depositNativeAmount],
                     )
                     .to.changeTokenBalances(iexecPoco, [accountA], [depositAmount])
                     .to.emit(iexecPoco, 'Transfer')
                     .withArgs(AddressZero, accountA.address, depositAmount);
             });
 
+            it('Should deposit native tokens and return excess', async () => {
+                const depositAmountExecess = 0.1;
+                const depositAmountWithExecess = 100.1;
+                const expectedTokenTransfer = depositAmountWithExecess - depositAmountExecess;
```

Comment:
No need for `expectedTokenTransfer` since its the same as `depositAmount`.
```suggestion
                const depositRemainder = 0.1;
                const depositAmountWithRemainder = depositAmount.add(depositRemainder);
```